test-cases/tests/performance/k03-run-performance-test-against-rhsso.md

```diff
@@ -32,17 +46,32 @@ You should have an AWS IAM account in [QE-AWS account](https://068334777414.sign
    - for example, Keycloak 10 = RHSSO 7.4
 5. `cd testsuite/performance/`
 6. Run `mvn clean install`
-7. Replace the username, password and server properties of `tests/target/provisioned-system.properties` with your clusters RHSSO details.
 
-   > keycloak.frontend.servers= `<rhsso-url>`
+   > you might need to delete/move the `tests/target/provisioned-system.properties` file for the command to pass. Once finished, put the file back.
+
+7. Replace the username, password and server properties of `tests/target/provisioned-system.properties` with your clusters RHSSO details. Make sure `/auth` is appended at the end of RHSSO URL.
+
+   > keycloak.frontend.servers= `<rhsso-url>/auth`
    >
    > keycloak.admin.user= `<admin-user>`
    >
    > keycloak.admin.password= `<admin-pwd>`
 
-8. Run `mvn verify -Pgenerate-data -Ddataset=1r_10c_100u -DnumOfWorkers=10 -Djackson.version=2.10.1`
+8. Run `mvn verify -Pgenerate-data -Ddataset=1r_10c_100u -Djackson.version=2.10.1`
+
+   > this should create `realm_0` realm and 100 users in that realm. If both realm and users already exists, this can be skipped. It is idempotent, running it more times makes no harm though.
+
    - **NOTE:** `-Djackson.version=2.10.1` is a workaround for RHSSO 7.4 only. Please remove this param from the test case when we upgrade to RHSSO 7.5
-9. Run
+
+9. Execute single performance test
+
+   `mvn verify -Ptest -Ddataset=1r_10c_100u`
+
+   > This is just to verify that everything is ok before running the actuall stress test. If the command passes ok you can proceed with the next step.
+
+   > If experiencing 404 not found issues, double check the RHSSO URL, make sure the `/auth` is appended. It might also be required to set Java Trust Store using `-DtrustStore=PATH_TO/TrustStore.jks -DtrustStorePassword=<password-if-it-is-password-protected>`.
```

Comment:
May I know from where can I get this .jks file

Reply:
Ah, it is apparently not clear enough. There's new subsection called "How to create Trust Store JKS file"  withing "Prerequisities" section describing how to create that JKS file

Reply:
I guess I need to add something like `See "How to create Trust Store JKS file" section for details on how to create the JKS file.`

Reply:
Ah, nvm. my bad reading habit, I did not see the subsection under pre-requisite, I think that's enough.